web/pages/[username]/[contractSlug].tsx

```diff
@@ -54,25 +51,14 @@ export const getStaticProps = fromPropz(getStaticPropz)
 export async function getStaticPropz(props: {
```

Comment:
Aside: I think we should kill off this 'getStaticPropz' pattern. We aren't running any private instances, I don't think we should be running any for the foreseeable future, and if we were, it would be better to use server side auth to keep the data private.

Reply:
Talk to Austin about it, he thinks it's still useful.